code/modules/bitrunning/util/virtual_megafauna.dm

```diff
@@ -6,8 +6,23 @@
 
 	true_spawn = FALSE
 
+	// rebuild the achievement element's arguments to remove it appropriately
+	if (achievement_type || score_achievement_type)
+		var/list/achievements = list(/datum/award/achievement/boss/boss_killer, /datum/award/score/boss_score)
+		if (achievement_type)
+			achievements += achievement_type
+		if (score_achievement_type)
+			achievements += score_achievement_type
+		RemoveElement(/datum/element/kill_achievement, string_list(achievements), crusher_achievement_type, /datum/memory/megafauna_slayer)
```

Comment:
I guess it is much easier to kill them within the domain than on lavaland, and less risky too, but please changelog this change as well